server/svix-server/src/lib.rs

```diff
@@ -135,3 +136,28 @@ pub async fn run(cfg: Configuration, listener: Option<TcpListener>) {
     worker_loop.expect("Error initializing worker");
     expired_message_cleaner_loop.expect("Error initializing expired message cleaner")
 }
+
+mod docs {
+    use axum::{
+        response::{Html, IntoResponse, Redirect},
+        routing::get,
+        Json, Router,
+    };
+
+    pub fn router() -> Router {
+        Router::new()
+            .route("/", get(|| async { Redirect::permanent("/docs") }))
```

Comment:
302, so not permanent, but temporary (who knows, we may want to do a different redirect).